File: aoj/volume100/nodejs/A10007.js
function main(lines) {
  for (var i = 0;; i++) {
    if (lines[i] == '0 0')
      break;

    var nums = lines[i].split(/\s+/).sort(function(a, b) {
      return a - b;
    });
    console.log(nums.join(' '));
  }
}

var chunks = [];
process.stdin.on('data', function(chunk) {
  chunks.push(chunk)
});
process.stdin.on('end', function() {
  var lines = chunks.join('').split('\n');
  main(lines);
});

File: aoj/volume100/haskell/A10007.hs
import Control.Applicative
import Data.List

main = do
  inputs <- lines <$> getContents
  mapM_ putStrLn
    $ fmap unwords
    $ fmap (fmap show)
    $ fmap sort
    $ fmap (fmap (read :: String -> Int) . words)
    $ takeWhile (/="0 0") inputs
